chess/polyglot.py

```diff
@@ -65,9 +72,10 @@ def __init__(self, filename):
 
         try:
             self.mmap = mmap.mmap(self.fd, 0, access=mmap.ACCESS_READ)
-        except ValueError:
+        except Exception as e:
```

Comment:
Let's just catch both like `except (ValueError, mmap.error):` and not look closer at `e.errno`, if we can't handle it differently anyway.